### PR TITLE
Fixed the blank line parsing issue.

### PR DESCRIPTION
**Pull request by Abhijithang**
Added a new exception class to handle if htpasswd file contains blank lines.

### CODE REVIEW

**Comment by qk4l** [comment]
Hi, just got the problem with htpasswd file which was ends blank line.

**Comment by Abhijithang** [comment]
Hi, Fixed the parsing issue which was causing due to blank lines.

**Comment by qk4l** [comment]
@k0st1an 
Could you merge it, please ?

File: htpasswd/basic.py [hunk 1]
```diff
@@ -8,7 +8,6 @@
 except ImportError:
     from ordereddict import OrderedDict
 
-
```
Comment (thesharp) [inline htpasswd/basic.py]: This blank line was here for a reason, don't remove it.

Comment (Abhijithang) [reply htpasswd/basic.py]: Sure, I wasn't aware of it. sorry.  :)

**Pull request merged**